src/functions.jl

```diff
@@ -102,7 +108,7 @@ Call a polymake method on the object `obj` with the given `func` name and given
 If `void=true` the function is called in a void context. For example this is important for visualization.
 """
 function call_method(obj, func::Symbol, args...; void=false, unwrap=true, kwargs...)
-    fname = string(func)
+    fname = uppercase(string(func))
```

Comment:
I am not sure this actually works. For example `faces_of_dim` is a user method.

Reply:
true, I'll fix that